### PR TITLE
Require setuptools

### PR DESCRIPTION
**Pull request by lukaszachy**
Library is used by python3-tmt so we need to require it, it is not guaranteed to be present on the system.

### CODE REVIEW

**Comment by lukaszachy** [comment]
@happz proposed better way to add setuptools into 'setup.py', I'll amend change that way.

File: tests/prepare/basic/data/main.fmf [hunk 1]
```diff
@@ -3,7 +3,7 @@ provision:
     image: fedora
 prepare:
     how: install
-    package: tmt
+    package: fmf
 execute:
     how: tmt
-    script: tmt --help
+    script: fmf --help
```
Comment (happz) [inline tests/prepare/basic/data/main.fmf:9]: Do these changes look unrelated, or am I missing something?

Comment (lukaszachy) [reply tests/prepare/basic/data/main.fmf:9]: Without this change this test will keep failing until fixed tmt rpm reaches stable repositories.
IMO fmf is good enough to test that package installation works (or any other package for that manner)

Comment (lukaszachy) [reply tests/prepare/basic/data/main.fmf:9]: Ah, we assert the output as well, need to amend that as well.

Comment (lukaszachy) [reply tests/prepare/basic/data/main.fmf:9]: Fixed in a14e15fe

Comment (happz) [reply tests/prepare/basic/data/main.fmf:9]: Understood.

**Review by happz — APPROVED** [review]
